core/lib/web3_decl/src/error.rs

```diff
@@ -1,5 +1,18 @@
 //! Definition of errors that can occur in the zkSync Web3 API.
 
+use std::{
+    collections::HashMap,
+    error,
+    error::Error,
+    fmt,
+    future::Future,
+    mem,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use jsonrpsee::core::ClientError as RpcError;
```

Comment:
Nit: This alias contributed to the confusion, tbh. jsonrpsee has a lot of error types, and I didn't really understand which error type it was.